next.config.js

```diff
@@ -75,7 +75,16 @@ module.exports = withHashicorp({
 			JSON.stringify(simpleRedirects, null, 2),
 			'utf-8'
 		)
-		return complexRedirects
+		return (
+			[...complexRedirects],
+			[
+				{
+					source: '/:path*/index',
+					destination: '/:path*',
+					permanent: true,
```

Comment:
There was a discussion about how this was possibly a temporary fix and there was a better solution that was to be investigated. As is would have a preference for putting the redirect config in one location such as https://github.com/hashicorp/dev-portal/blob/main/build-libs/redirects.js so that config lives in one location.

Reply:
The changes I mentioned in our meeting were in commits that I squashed related to [this block of code](https://github.com/hashicorp/dev-portal/blob/main/src/lib/docs/add-branded-overview-sidebar-item.ts#L12-L19). 

This redirect is an attempt to prevent links ending in `/index` in docs (or tutorials for that matter) from 404ing

Reply:
fixed in https://github.com/hashicorp/dev-portal/pull/2542/commits/538d888d2b00ab17b6a17dd6d4e3c7543fd16c5e